src/cursor.ts

```diff
@@ -3,21 +3,46 @@ import {
   ReferenceExpression,
   SelectQueryBuilder,
   StringReference,
+  sql,
 } from "kysely";
 
+declare const SIMPLE_COLUMN_DATA_TYPES: readonly [
```

Comment:
I copied this from `kysely` and stripped it a little bit. So we only need to support the "standard" types for nullable field pagination.

I think this might potentially still be a bit too long. See the `minMaxValues` function at the bottom for how this is used